src/FileCabinet/SuiteScripts/TransferOrderV2/HC_MR_ExportedWhtoStoreTOJson.js

```diff
@@ -0,0 +1,289 @@
+/**
+ * @NApiVersion 2.1
+ * @NScriptType MapReduceScript
+ */
+define(['N/error', 'N/file', 'N/task', 'N/record', 'N/search', 'N/sftp'],
+ 
+    (error, file, task, record, search, sftp) => {
+        const internalIdList = new Set([]);
+        
+        const checkInternalId = (internalid) => {
+            if (internalIdList.has(internalid)) {
+                return false;
+            } else {
+                internalIdList.add(internalid);
+                return true;
+            }
+        }
+
+        const getInputData = (inputContext) => { 
+            // Get StoreTransferOrder search query
+            var StoreTransferOrderSearch = search.load({ id: 'customsearch_hc_exp_wh_to_store_to_v2' });
+            return StoreTransferOrderSearch
```

Comment:
StoreTransferOrderSearch -> WhToStoreTransferOrderSearch
also update the comment